action.yaml

```diff
@@ -0,0 +1,49 @@
+name: Elixir Build and Cache Dependencies
+inputs:
+  elixir-version:
+    required: true
+    type: string
+  erlang-version:
+    required: true
+    type: string
+  mix-env:
+    required: true
+    type: string
+outputs:
+  cache-key:
+    value: ${{ steps.compute-cache-key.outputs.cache-key }}
+  cache-key-suffix:
+    value: ${{ steps.compute-cache-key.outputs.cache-key-suffix }}
+runs:
+  using: composite
+  steps:
+    - uses: actions/checkout@v2
+    - uses: erlef/setup-elixir@v1
+      with:
+        elixir-version: ${{ inputs.elixir-version }}
+        otp-version: ${{ inputs.erlang-version }}
+    - name: Compute Cache Key
+      id: compute-cache-key
+      run: |
+        elixir="elixir-${{ inputs.elixir-version }}"
+        erlang="erlang-${{ inputs.erlang-version }}"
+        mix_env="${{ inputs.mix-env }}"
+        runner="${{ runner.os }}"
+        suffix="${runner}_${erlang}_${elixir}_${mix_env}_${{ hashFiles('mix.lock') }}"
```

Comment:
I'm not certain `runner` is needed as part of the cache key, but I guess there is no harm in keeping it.

Reply:
I had the same thought. It is not clear that `runner` is needed for Elixir/Erlang, but I added it, because other examples seem to use it.